src/components/30-organisms/table/README.md

```diff
@@ -0,0 +1,57 @@
+# Table
```

Comment:
Do we have to define code in here? IMHO that can get outdated very fast and this is very annoying to any users who want to implement this table and this the documentation is always "real". We are consisten everywhere else, but if we happen to get into such a situation here, the users might think that the documentation might be outdated everywhere else too.

Reply:
Yeah i agree.. so we need to make a mechnaism that reads react code and previews it as atm my addon does only wc code